<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRMapWidget</class>
 <widget class="QWidget" name="VRMapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRMapWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#leftWidget{
background-color:#1f2020;
border-right:1px solid #111111;
}
#label{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#868989;
line-height:12px;
text-align:left;
}
#line{
background:#444444;
}
#listWidget{
background-color:#1f2020;
border:none;
}
QListView::item{
padding-left:15px;
}
QListView::item:hover{
/*border:1px solid #00A092;*/
}
QToolButton,
QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:rgb(255,255,255);
border-radius:15px;
background:rgba(0,0,0,0.7);
}
#mapSettingBtn{
qproperty-icon:url(:/image/image/mapSet.png);
qproperty-iconSize:18px 18px;
padding-left:4px;
}
#mapSettingBtn::hover{
border: 2px solid #8f8f91;
border-radius:15px;
}
#mapSettingBtn:pressed{
border-radius:14px;
margin:1px;
}
#pathBuildingBtn{
qproperty-icon:url(:/image/image/pathBuild.png);
qproperty-iconSize:18px 18px;
padding-left:4px;
}
#pathBuildingBtn::hover{
border: 2px solid #8f8f91;
border-radius:15px;
}
#pathBuildingBtn:pressed{
border-radius:14px;
margin:1px;
}
#addPanoramaBtn::hover{
color:#FFD500;
}
#initPage{
background-color:#1f2020;
}
#label{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:rgba(238,238,238,1);
}
#addMapBtn{
border-radius:14px;
border-image:url(:/image/image/addMap.png);
}
#addMapBtn::hover{
border-image:url(:/image/image/addMaph.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QToolButton" name="addPanoramaBtn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>+ 添加全景</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgba(17,17,17,1);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="commonListWidget" name="listWidget">
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::CopyAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="initPage">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>55</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>9</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>72</width>
                <height>17</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>72</width>
                <height>17</height>
               </size>
              </property>
              <property name="text">
               <string>点击添加地图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addMapBtn">
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="MapWgt" name="mapPage"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>commonListWidget</class>
   <extends>QListWidget</extends>
   <header>commonListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>MapWgt</class>
   <extends>QWidget</extends>
   <header>MapWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
